integration/token/interop/tests.go

```diff
@@ -149,18 +149,24 @@ func TestHTLCNoCrossClaimTwoFabricNetworks(network *integration.Infrastructure)
 	tmsIssueCash(network, beta, "issuer", "", "USD", 30, "bob.id1")
 	checkBalance(network, "bob", "bob.id1", "USD", 30, token.WithTMSID(beta))
 
-	preImage, hash := htlcLock(network, alpha, "alice", "alice.id1", "EUR", 10, "alice.id2", 30*time.Second, nil, 0)
-	htlcLock(network, beta, "bob", "bob.id1", "USD", 10, "bob.id2", 30*time.Second, hash, 0)
+	aliceLockTxID, preImage, hash := htlcLock(network, alpha, "alice", "alice.id1", "EUR", 10, "alice.id2", 30*time.Second, nil, 0)
+	bobLockTxID, _, _ := htlcLock(network, beta, "bob", "bob.id1", "USD", 10, "bob.id2", 30*time.Second, hash, 0)
 
 	go func() { htlcClaim(network, alpha, "alice", "alice.id2", preImage) }()
 	go func() { htlcClaim(network, beta, "bob", "bob.id2", preImage) }()
-	scan(network, "alice", hash, crypto.SHA256, token.WithTMSID(alpha))
-	scan(network, "bob", hash, crypto.SHA256, token.WithTMSID(beta))
+	scan(network, "alice", hash, crypto.SHA256, "", token.WithTMSID(alpha))
```

Comment:
Why are we even using scan in this test?

Reply:
just to check the different type of scan works as expected.